BepInEx.Core/Configuration/ConfigFile.cs

```diff
@@ -217,6 +229,63 @@ public ConfigEntryBase[] GetConfigEntries()
                 return Entries.Values.ToArray();
         }
 
+        /// <summary>
+        /// Get the value of an orphaned entry.
+        /// </summary>
+        /// <param name="entry">The entry to try and retrieve.</param>
+        /// <param name="val">Out value of this entry (if it exists).</param>
+        /// <returns>True if the entry exists in the dictionary.</returns>
+        public bool TryGetOrphanedEntry(ConfigDefinition entry, out string val)
```

Comment:
It would be nice if the methods were generic instead of returning `string`.
Deserialization could happen on demand (with caching).

Reply:
848c405d20f83424050e0d4a85d57971eb5949e8